pkg/probe/tcp/tcp.go

```diff
@@ -21,9 +21,8 @@ import (
 	"strconv"
 	"time"
 
-	"k8s.io/kubernetes/pkg/probe"
-
 	"k8s.io/klog/v2"
```

Comment:
the intent was to keep imports from this repo together... k8s.io/klog is an external repo

Reply:
i think all packages of having prefix `k8s.io/` be better put together